dotcom-rendering/src/model/enhance-H2s.ts

```diff
@@ -78,10 +87,11 @@ const enhance = (elements: CAPIElement[]): CAPIElement[] => {
 	return enhanced;
 };
 
-export const enhanceH2s = (blocks: Block[]): Block[] =>
-	blocks.map((block: Block) => {
+export const enhanceH2s = (blocks: Block[]): Block[] => {
+	return blocks.map((block: Block) => {
```

Comment:
Do we need this return?

Reply:
No. But there are _lots_ of examples of this pattern in the codebase and I don't think DCR has (or should have) a rule banning needless `return`s.

The argument for them is that they provide a more explicit statement for what is happening. The argument against them is that the code will be shorter and more concise. The language allows developers to choose which approach they take and I think we should keep this choice open rather than be too prescriptive for things like this.

Reply:
I actually think we _should be prescriptive_. Consistency is helpful as it ensures exposure to patterns.

If a developer is unfamiliar with arrow functions, I think they might be wondering what the difference is between the two approaches. I know I did when I first encoutered them. There’s an ESLint rule called [`arrow-body-style`](https://eslint.org/docs/latest/rules/arrow-body-style).

Reply:
I do think that lint rules and prettier are great things and generally support their use as a way to avoid bike-shedding around formatting but I think we need to be careful as seeing them as a solution for all formatting disagreements. I think that there are sometimes reasons why a developer might choose one approach over the other and we need to be careful to preserve those developer intentions.

In general, the two styles are very similar:

```js
// Implicit return
const add = (num1, num2) => num1 + num2
```

vs

```js
// Explicit return
const add = (num1, num2) => {
    return num1 + num2
}
```

In principle, forcing the code to use one approach over the other makes little difference but I find one of the disadvantages of some lint rules is they will override intentional formatting and can sometimes make, longer, more complex sequences of code, messy. I think we've all experienced that disappointment when you've written a lovely block of beautiful code and then you hit save and Prettier mangles it 😢 . My fear here is that there are places in DCR where this might happen with this particular lint rule

Reply:
Also, yes, happy to have this as a separate discussion to unblock this PR